### PR TITLE
separate c/h files for new encoder

### PR DESCRIPTION
**Pull request by cbrugal**
-Separated the c/h
-Changed the output length finder to the same lib's length finder

### CODE REVIEW

File: src/pch.h [hunk 1]
```diff
@@ -22,6 +22,7 @@
 
 #include "e_akv_err.h"
 #include "base64.h"
```
Comment (polyakov-vlad) [inline src/pch.h:24]: This needs to be deleted - we don't need the old encoder

File: src/base64lib.c [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    64, 64, 64, 64, 64, 64, 64, 64, 64, 64, 64, 64, 64, 64, 64, 64,
+    64, 64, 64, 64, 64, 64, 64, 64, 64, 64, 64, 64, 64, 64, 64, 64,
+    64, 64, 64, 64, 64, 64, 64, 64, 64, 64, 64, 64, 64, 64, 64, 64,
+    64, 64, 64, 64, 64, 64, 64, 64, 64, 64, 64, 64, 64, 64, 64, 64,
+    64, 64, 64, 64, 64, 64, 64, 64, 64, 64, 64, 64, 64, 64, 64, 64
+};
+
+static const char basis_64[] = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/";
+
+int Base64encode_len(int len) {
```
Comment (polyakov-vlad) [inline src/base64lib.c]: Not a C style name; please fix

Comment (polyakov-vlad) [inline src/base64lib.c]: Add comments including input param documentation. Please refer to comments format in Roxy repository

File: src/base64lib.c [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    return ((len + 2) / 3 * 4) + 1;
+}
+
+void Base64encode(char *encoded, const char *string, int len) {
```
Comment (polyakov-vlad) [inline src/base64lib.c]: Please add method comments

**Review by polyakov-vlad — CHANGES_REQUESTED** [review]

File: src/pch.h (hunk 1 above)
Comment (cbrugal) [reply src/pch.h:24]: May need to check up on this- for some reason the way the engine is built, all headers have to be here or else it fails to compile.

**Review by polyakov-vlad — APPROVED** [review]

**Pull request merged**